File: Haskell/beginner/1175.hs
import Control.Monad.RWS.CPS (MonadState(put))
-- https://judge.beecrowd.com/pt/problems/view/1175
-- Troca em Vetor I

-- Main entry point
main :: IO ()
main = do
    -- Read 20 integers from input
    usrVector <- sequence [readLn :: IO Int | _ <- [1..20]]

    -- Reversing the vector
    let reversedVector = reverse usrVector

    -- Print the reversed vector with the specified format
    mapM_ (\(i, v) -> putStrLn $ "N[" ++ show i ++ "] = " ++ show v) (zip [0..] reversedVector)

-- By: Fernando Serra (Thanks, Washu-sama!)
-- https://github.com/fernandosserra

File: Python/beginner/1175.py
# https://judge.beecrowd.com/pt/problems/view/1175
# Troca em Vetor I

# Getting inputs for the first vector
usrvector = [int(input()) for _ in range(20)]

# Reversing the vector
for i in range(10):
    usrvector[i], usrvector[19 - i] = usrvector[19 - i], usrvector[i]
    
# Printing the reversed vector
for i in range(20):
    print(f"N[{i}] = {usrvector[i]}")
    
# By: Fernando Serra
# https://github.com/fernandosserra